### ISSUE TITLE
extra WS in @brief docu block break layout

### ISSUE THREAD

**Issue by aotto1968** [doxygen/doxygen#11031]
**Describe the bug**
I get an extra **WS** (WhiteSpace) which I have **not** added to code.

The test case create the same docu string 2 times :
1. as direct string on the @page → right
2. as @brief docu string → wrong

the string is `(\arg{in}|\arg{out}|\arg{inout})` a selection of the items *in*, *out* and *inout* connected with `|` in an @arg alias.

1. The @arg alias is defined as `ALIASES += "arg{1}=<b><em>\1</em></b>"`

**Screenshots**
1. RIGHT
![image](https://github.com/user-attachments/assets/100ba5c4-fc5b-40bc-a595-b9063f63b53c)

2. WRONG
![image](https://github.com/user-attachments/assets/5ae6ea4b-f86f-41b7-9658-183bfccde735)

**To Reproduce**
use attachment
[doxygen_bug__extra_WS_break_layout.tar.gz](https://github.com/user-attachments/files/16390427/doxygen_bug__extra_WS_break_layout.tar.gz)

**Expected behavior**
no WS

**Version**
> doxygen -V
1.12.0 (7673d22af1cea4d84bdc00164d9c9a25d693a714)
    with sqlite3 3.42.0.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #11032

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by albert-github** [comment]
@doxygen 
Looks like that the internal documentation goes in an endless loop due to the introduced optimization 
```
Commit: 9b97d4f516878a87e4180547189d9d77bf401212 [9b97d4f]

Date: Thursday, August 1, 2024 4:43:16 PM

Change lexer rules to avoid the need for RLopt1
```

Rule: 
```
--accepting rule at line 1064 (" \ilinebr ")
```

previous output lines as reference:
```
--accepting rule at line 440 ("// no need for \ilinebr here as the previous line was empty and was skipped
")
--accepting rule at line 1064 (" no need for \ilinebr ")
--accepting rule at line 1064 (" \ilinebr ")

```

**Comment by albert-github** [comment]
The problem reported in https://github.com/doxygen/doxygen/issues/11031#issuecomment-2263491797has been fixed in  7d4a9d188444c143995140eb7eeeaea019d13b6a

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.12.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
